lib/iris/cube.py

```diff
@@ -2221,7 +2221,7 @@ def new_cell_measure_dims(cm_):
         data = copy.deepcopy(data)
 
         # We can turn a masked array into a normal array if it's full.
-        if isinstance(data, ma.core.MaskedArray):
```

Comment:
`ma.core.MaskedArray`!! there are quite a few different ways this has been done.